src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,23 @@
+.catalog {
+  max-width: 944px;

```

Comment:
you shouldn't hardcode max-width
it should be calculated according to the width and number of the cards